file-formats/intf/zif_aff_types_v1.intf.abap

```diff
@@ -1,20 +1,15 @@
-INTERFACE zif_aff_types_v1 PUBLIC .
-
+INTERFACE zif_aff_types_v1 PUBLIC.
 
   TYPES:
     "! Language
     language TYPE c LENGTH 2.
 
   TYPES:
-    "! ABAP language version
-    BEGIN OF ENUM abap_language_version ,
-      "! Standard
+    BEGIN OF ENUM abap_language_version,
       standard VALUE IS INITIAL,
-      "! ABAP cloud development
```

Comment:
Why did you remove the ABAP doc?

Reply:
No, sorry, I missed somehow that I didn't check the final version. 
It's perfectly fine for me if the last version is correct.